receiver/sqlserverreceiver/scraper.go

```diff
@@ -470,3 +490,258 @@ func (s *sqlServerScraperHelper) recordDatabaseStatusMetrics(ctx context.Context
+		}
+
+		obfuscatedStatement, err := obfuscateSQL(row[statementText])
+		if err != nil {
+			s.logger.Error(fmt.Sprintf("failed to obfuscate SQL statement value: %s err: %s", row[statementText], err))
+		}
+
+		record := scopedLog.LogRecords().AppendEmpty()
+		record.SetTimestamp(pcommon.NewTimestampFromTime(time.Now()))
+		record.Attributes().PutStr("db.namespace", row[DBName])
+		record.Attributes().PutStr("network.peer.address", row[clientAddress])
+		record.Attributes().PutInt("network.peer.port", int64(clientPortNumber))
+		record.Attributes().PutStr(dbPrefix+queryStart, row[queryStart])
+		record.Attributes().PutInt(dbPrefix+sessionID, int64(sessionIDNumber))
+		record.Attributes().PutStr(dbPrefix+sessionStatus, row[sessionStatus])
+		record.Attributes().PutStr(dbPrefix+requestStatus, row[requestStatus])
+		record.Attributes().PutStr(dbPrefix+hostname, row[hostname])
+		record.Attributes().PutStr(dbPrefix+command, row[command])
+		// Following Opentelemetry Semantic Convention for this naming.
+		record.Attributes().PutStr("db.query.text", obfuscatedStatement)
+		record.Attributes().PutInt(dbPrefix+blockingSessionID, int64(blockingSessionIDNumber))
+		record.Attributes().PutStr(dbPrefix+waitType, row[waitType])
+		record.Attributes().PutInt(dbPrefix+waitTime, int64(waitTimeVal))
+		record.Attributes().PutStr(dbPrefix+waitResource, row[waitResource])
+		record.Attributes().PutInt(dbPrefix+openTransactionCount, int64(openTransactionCountVal))
+		record.Attributes().PutInt(dbPrefix+transactionID, int64(transactionIDVal))
+		record.Attributes().PutDouble(dbPrefix+percentComplete, percentCompleteVal)
+		record.Attributes().PutDouble(dbPrefix+estimatedCompletionTime, estimatedCompletionTimeVal)
+		record.Attributes().PutInt(dbPrefix+cpuTime, int64(cpuTimeVal))
+		record.Attributes().PutInt(dbPrefix+totalElapsedTime, int64(totalElapsedTimeVal))
+		record.Attributes().PutInt(dbPrefix+reads, int64(readsVal))
+		record.Attributes().PutInt(dbPrefix+writes, int64(writesVal))
+		record.Attributes().PutInt(dbPrefix+logicalReads, int64(logicalReadsVal))
+		record.Attributes().PutInt(dbPrefix+transactionIsolationLevel, int64(transactionIsolationLevelVal))
+		record.Attributes().PutInt(dbPrefix+lockTimeout, int64(lockTimeoutVal))
+		record.Attributes().PutInt(dbPrefix+deadlockPriority, int64(deadlockPriorityVal))
+		record.Attributes().PutInt(dbPrefix+rowCount, int64(rowCountVal))
+		record.Attributes().PutStr(dbPrefix+queryHash, queryHashVal)
+		record.Attributes().PutStr(dbPrefix+queryPlanHash, queryPlanHashVal)
+		record.Attributes().PutStr(dbPrefix+contextInfo, contextInfoVal)
+
+		record.Attributes().PutStr(dbPrefix+username, row[username])
+
+		waitCode, waitCategory := getWaitCategory(row[waitType])
+		record.Attributes().PutInt(dbPrefix+"wait_code", int64(waitCode))
+		record.Attributes().PutStr(dbPrefix+"wait_category", waitCategory)
```

Comment:
Where is this `wait_code` and `wait_category` coming from? Could you give me the documentation link regarding these two values? And, why do we need to mapping these?

Reply:
i think this is from our internal requirements